Makefile

```diff
@@ -165,8 +148,8 @@ vendor:
 build: gen vet lint
 	${GO} build -buildvcs=false
 
-.PHONY: build/odh
-build/odh: vet
+.PHONY: build/docker
+build/docker: vet
 	${GO} build -buildvcs=false
```

Comment:
Can we do without `-buildvcs` now, or do we still need this option?

Reply:
I managed to build it without this flag, let's see if that works on the CI as well.